docker/rucio_client/scripts/cmslinks.py

```diff
@@ -235,10 +251,18 @@ def update(self, overwrite=False, disable=True, dry=False, srcselect=r'\S+', dst
 
     OPTIONS = PARSER.parse_args()
 
+    # Configure logger
+    # Add new logging level, so usefull info is seperated from info
+    USEFULL_INFO_NUM = 25
```

Comment:
Why do you need a custom logging level?

Reply:
In the script there are info and warning logs. By default logging level is set to info, however info logs mainly populate the logs instead of providing helpful information. In addition, there is information that there is need to be logged, for example when a new distance is set or when a distance is overwritten. So, an new log level, that sits between info and warning levels, was set for this king of information. This new level was set also as default sο that there is no unneeded filling of the log files.

Reply:
Feel free to remove/modify the level for existing logs. I believe the 5 default log levels are enough.

Reply:
Unimportant logs moved to debug level